recipes-openxt/tss/tpm2-tss_git.bb

```diff
@@ -3,21 +3,22 @@ DESCRIPTION = "tpm2-tss for interfacing with tpm2.0 device"
 SECTION = "tpm"
 
 LICENSE = "License-tss"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=500b2e742befc3da00684d8a1d5fd9da"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0b1d631c4218b72f6b05cb58613606f4"
 
 NO_GENERIC_LICENSE[License-tss] = "LICENSE"
 
-DEPENDS = "autoconf-archive autoconf pkgconfig"
+DEPENDS = "autoconf-archive autoconf pkgconfig libgcrypt"
 
 SRC_URI = " \
-    git://github.com/01org/tpm2-tss.git;protocol=git;branch=master;name=tpm2-tss;destsuffix=tpm2-tss \
+    git://github.com/01org/tpm2-tss.git;protocol=git;branch=master;nobranch=1;name=tpm2-tss;destsuffix=tpm2-tss \
```

Comment:
See `tpm2-tools_git.bb`, the same comment applies.

Reply:
Addressed.